src/util-file-swf-decompression.c

```diff
@@ -32,9 +32,9 @@
 #include "util-misc.h"
 #include "util-print.h"
 
-#include <zlib.h>
+#include "rust.h"
 
-#include <htp/lzma/LzmaDec.h>
+#include <zlib.h>
```

Comment:
Can the zlib.h include be removed?

Reply:
I don't think so. It is still used in the CWS decompression path.

Reply:
Ok, make sense.  Even if removed from this file, its pulled in via one of the other includes, so we shouldn't remove it.